### ISSUE TITLE
Fails to build with autoconf 2.71

### ISSUE THREAD

**Issue by lazka** [ianlancetaylor/libbacktrace#71]
```shell
> autoreconf -vfi
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I config
configure.ac:33: error: Please use exactly Autoconf 2.69 instead of 2.71.
config/override.m4:12: _GCC_AUTOCONF_VERSION_CHECK is expanded from...
configure.ac:33: the top level
autom4te: error: /usr/bin/m4 failed with exit status: 1
aclocal-1.15: error: echo failed with exit status: 1
autoreconf: error: aclocal failed with exit status: 1
```

If that's not supported feel free to close.

**Comment by ianlancetaylor** [comment]
This is working as intended.  libbacktrace uses the exact configury versions as used by GCC.

**Comment by lazka** [comment]
ok, thanks, I'll patch this downstream then.

**Comment by ianlancetaylor** [comment]
I've reconsidered this.  While the version of libbacktrace in the GCC sources should require autoconf 2.69, there's no special reason to require 2.69 in the standalone libbacktrace.  I've commented out the code (copied from GCC) that requires exactly 2.69.

That said using autoconf 2.71 reports some warnings about obsolete autoconf commands.  I don't plan to fix those unless and until GCC starts avoiding them.
